osquery/tables/utility/osquery.cpp

```diff
@@ -229,6 +229,15 @@ QueryData genOsqueryInfo(QueryContext& context) {
     r["watcher"] = "-1";
   }
 
+  std::string uuid;
+  r["uuid"] = (getHostUUID(uuid)) ? uuid : "";
+
+  std::string ephemeral;
+  r["ephemeral_id"] = (getEphemeralUUID(ephemeral)) ? ephemeral : "";
```

Comment:
This wont report the expected ephemeral ID, it will report a new UUID each time the table is called.

Perhaps remove reporting this via the `osquery_info` table?